frontend/src/components/cluster/Overview.tsx

```diff
@@ -26,13 +26,11 @@ const useOverviewStyle = makeStyles({
 });
 
 export default function Overview() {
-  const [pods, setPods] = React.useState<Pod[] | null>(null);
-  const [nodes, setNodes] = React.useState<Node[] | null>(null);
   const { t } = useTranslation(['translation']);
   const classes = useOverviewStyle();
 
-  Pod.useApiList(setPods);
-  Node.useApiList(setNodes);
+  const [pods] = Pod.useList();
```

Comment:
The useList API also returns a error in case something went wrong while making the request, should we handle it here??

Reply:
We could do that in a new PR if needed. This logic should do the same as the one it's changing (minus returning null again if the cluster changes).